Command/ClearExpiredTokensCommand.php

```diff
@@ -26,31 +27,44 @@ final class ClearExpiredTokensCommand extends Command
      */
     private $refreshTokenManager;
 
+    /**
+     * @var AuthorizationCodeManagerInterface
+     */
+    private $authorizationCodeManager;
+
     public function __construct(
         AccessTokenManagerInterface $accessTokenManager,
-        RefreshTokenManagerInterface $refreshTokenManager
+        RefreshTokenManagerInterface $refreshTokenManager,
+        AuthorizationCodeManagerInterface $authorizationCodeManager
     ) {
         parent::__construct();
 
         $this->accessTokenManager = $accessTokenManager;
         $this->refreshTokenManager = $refreshTokenManager;
+        $this->authorizationCodeManager = $authorizationCodeManager;
     }
 
     protected function configure(): void
     {
         $this
-            ->setDescription('Clears all expired access and/or refresh tokens')
+            ->setDescription('Clears all expired access and/or refresh tokens and/or auth codes')
             ->addOption(
-                'access-tokens-only',
+                'access-tokens',
                 'a',
                 InputOption::VALUE_NONE,
-                'Clear only access tokens.'
+                'Clear expired access tokens.'
             )
             ->addOption(
-                'refresh-tokens-only',
+                'refresh-tokens',
                 'r',
                 InputOption::VALUE_NONE,
-                'Clear only refresh tokens.'
+                'Clear expired refresh tokens.'
+            )
+            ->addOption(
+                'auth-codes',
+                'ac',
```

Comment:
@X-Coder264 I don't think shortcuts can have more than one letter, I've tested this & got
```
  The "-c" option does not exist.
```

Reply:
@HypeMC Fixed.